.github/workflows/python-app.yml

```diff
@@ -26,6 +32,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8 pytest mypy pylint coverage
+        if [ -f actions-requirements.txt ]; then pip install -r actions-requirements.txt; fi
```

Comment:
what is the advantage of defining two requirements files?

Reply:
It is a kind of workaround. By default, pip reinstalls packages, which are built from the local directory, no matter what version is installed already. As this requirements file contains git reference to dispatcher master, it will rewrite the previously installed dispatcher in  https://github.com/oda-hub/dispatcher-container. 
If all submodules there are latest master - it is not a problem. 
But if you want to fix the dispatcher commit used in container - it is a problem.

Another option is to fix the actual commit in requirements.txt itself. But it is hard to maintain. 
That's why I moved it completely into another file as a precaution

Reply:
I appreciate the problem, but there are more conventional mechanisms to address this challenge, for example using `install_requires` which is [meant to serve a bit different purpose than requirements.txt](https://packaging.python.org/discussions/install-requires-vs-requirements/). Or even better use `tests_require`.
I understand that current convention of dispatcher is to ignore this difference, but it's causing a problem in this case.